### DISCUSSION TITLE
Scan port 25,110 for false positives

### DISCUSSION THREAD

**Discussion by komomon** [projectdiscovery/naabu#573] [Q&A]
在测试一些ip的时候，实际ip 25,110 端口未开放，但是扫描却显示目标端口开发。
如下是nabbu的检测结果
![image](https://user-images.githubusercontent.com/52700174/188857135-5b5ab535-33db-453e-af7d-b3ee5f8550d3.png)
如下是在线端口扫描工具的结果
![image](https://user-images.githubusercontent.com/52700174/188857361-02970f90-e2bd-4fcc-9911-06490c463976.png)

**Comment by ehsandeep** [comment]
@komomon, thanks for reporting this, but target host IP and used command are required for the investigation as there is a passive scan option as well.

**Comment by komomon** [comment]
我使用的命令是：naabu.exe -top-ports 100 -silent -no-color -list xxx.txt -o xxx.txt，没有使用--passive

**Comment by ehsandeep** [comment]
@komomon thank you, now we require host information to confirm/validate this behavior; if it's something that can not be shared here, feel free to jump into the discord server https://discord.gg/projectdiscovery and message to `pdteam`, thank you.

**Comment by akpioneer** [comment]
因为你电脑中的杀毒软件正在运行 例如火绒 
https://www.huorong.cn/
